source/upgrade-guide/troubleshooting.rst

```diff
@@ -53,7 +166,9 @@ This warning indicates that the Vulnerability Detection module might be disabled
 Application Not Found
 ---------------------
 
-If you encounter the message *Application Not Found* when accessing the Wazuh dashboard after upgrading, it might be that the configuration file ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` wasn't overwritten with new changes. To resolve this issue, update the ``uiSettings.overrides.defaultRoute`` setting with the ``/app/wz-home`` value in the configuration file:
+If you encounter the message *Application Not Found* when accessing the Wazuh dashboard after upgrading, it may be because the configuration file ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` wasn’t updated with the latest changes.
```

Comment:
```suggestion
If you see the message *Application Not Found* when accessing the Wazuh dashboard after upgrading, it may be because the configuration file ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` wasn’t updated with the latest changes.
```